### PR TITLE
travis test for parameter server

### PR DESCRIPTION
**Pull request by yangryan0**

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -4,7 +4,7 @@ AUTOMAKE_OPTIONS = foreign
 
 CXX = g++
 
-bin_PROGRAMS = parameter_server ps_test csv_to_libsvm $(TOP_DIR)/tests/test_travis/test_lr
+bin_PROGRAMS = parameter_server ps_test csv_to_libsvm $(TOP_DIR)/tests/test_travis/test_lr $(TOP_DIR)/tests/test_travis/test_ps $(TOP_DIR)/tests/test_travis/worker $(TOP_DIR)/tests/test_travis/error
```
Comment (jcarreira) [inline src/Makefile.am]: Can we break this line?

File: tests/test_travis/error.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  SparseDataset test_data = input.read_input_criteo_kaggle_sparse(
+      "tests/test_data/test_lr.csv", ",", config);
+  SparseLRModel model(1 << config.get_model_bits());
+
+  uint64_t start_time = get_time_us();
+
+  std::cout << "[ERROR_TASK] Computing accuracies"
+            << "\n";
+  FEATURE_TYPE avg_loss = 0;
+  for (int i = 0; i < 100; i++) {
```
Comment (jcarreira) [inline tests/test_travis/error.cpp]: Shouldn't we make this an infinite loop?

File: tests/test_travis/test.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for cmd in "timeout 60 ./tests/test_travis/test_ps" "timeout 50 ./tests/test_travis/worker" "timeout 50 ./tests/test_travis/worker" "./tests/test_travis/error"; do
```
Comment (jcarreira) [inline tests/test_travis/test.sh]: What happens when the loss doesn't go down or one of the workers (or PS) crashes? I'm not sure travis is going to fail in that case.
